openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -101,11 +101,7 @@ def get_object(self):
             )
 
         overview.enrollment = {'mode': mode, 'is_active': is_active}
-        if not is_active:
-            user_has_access = allow_public_access(overview, [COURSE_VISIBILITY_PUBLIC])
```

Comment:
This appears to be the only reference to this function, so we should delete the import above.